tools/releases/dockerfiles/Dockerfile.kuma-dp

```diff
@@ -0,0 +1,10 @@
+# using Envoy's base to inherit the Envoy binary
```

Comment:
Docker image should run under a non-`root` account, e.g. `nobody` (`65534:65534`) on `alpine`